readthedocs/embed/utils.py

```diff
@@ -15,13 +16,13 @@ def recurse_while_none(element):
     return {element.text: href}
 
 
-def clean_links(obj, url, html_raw_response=False):
+def clean_references(obj, url, html_raw_response=False):
     """
-    Rewrite (internal) links to make them absolute.
+    Rewrite (internal) links (href) and images (src) to make them absolute.
```

Comment:
I wonder if there's other external things we want to be rewriting here as well?

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/source is one example, perhaps.

Reply:
I created #9344  to track this and improve the API with other cases that may require rewriting the HTML content.